lib/src/assets/img/logo_nexus_generic.svg

```diff
@@ -0,0 +1 @@
+<svg width="72" height="72" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><g fill="#FFF"><path d="M54.162 46.032L35.617 57.03 17.322 46.467l.25-21.56 17.583-10.423-.064 5.29L22.065 27.5l-.19 16.266 13.804 7.97 13.991-8.298.179-15.248 4.553-2.701z"/><path d="M48.782 27.035l-12.155-7.008-.023-5.145 16.643 9.602z"/></g></g></svg>
```

Comment:
I think text formatting is still worthwhile.  And also some additional cleanup was possible.
```suggestion
<svg width="72" height="72" fill="#FFF" xmlns="http://www.w3.org/2000/svg">
  <path d="M54.162 46.032L35.617 57.03 17.322 46.467l.25-21.56 17.583-10.423-.064 5.29L22.065 27.5l-.19 16.266 13.804 7.97 13.991-8.298.179-15.248 4.553-2.701z"/>
  <path d="M48.782 27.035l-12.155-7.008-.023-5.145 16.643 9.602z"/>
</svg>

```